net/snort3/Makefile

```diff
@@ -21,15 +21,35 @@ PKG_CPE_ID:=cpe:/a:snort:snort
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
 
+ifneq ($(findstring powerpc,$(CONFIG_ARCH))$(findstring mips,$(CONFIG_ARCH)),)
+  CMAKE_OPTIONS += \
+	-DENABLE_TCMALLOC=OFF
+else
+  CMAKE_OPTIONS += \
+	-DENABLE_TCMALLOC=ON \
+	-DTCMALLOC_LIBRARIES=$(STAGING_DIR)/usr/lib/libtcmalloc.so
+  EXTRAGP:= +gperftools-runtime
+endif
+
+ifneq ($(findstring x86_64,$(CONFIG_ARCH))$(findstring aarch64,$(CONFIG_ARCH)),)
+  CMAKE_OPTIONS += \
+	-DENABLE_HYPERSCAN=ON \
+	-DHS_INCLUDE_DIRS=$(STAGING_DIR)/usr/include/hs
+  EXTRAVS:= +vectorscan-runtime
+else
```

Comment:
I was thinking about variants e.g. ``snort3-full`` or rather ``snort3-custom``. That will :100: % work, but in my opinion it was like... this package is heavily customized for you and only you will be using it even it will be available for others. 🤷

Reply:
I agree. Let's not make it more complex.